Code/MinimalLib/minilib.cpp

```diff
@@ -835,6 +835,39 @@ MCSResult getMcsResult(const JSMolList &molList,
 }
 } // namespace
 
+std::string get_mcs(const JSMolList &molList, const std::string &details_json) {
```

Comment:
I think it would be better to be consistent here and call this `get_mcs_as_json()`

Reply:
@greglandrum Done.